plugins/woocommerce-admin/client/order-attribution/style.scss

```diff
@@ -0,0 +1,61 @@
+.woocommerce-order-attribution-install-analytics-overview-banner {
+
+	margin: 0 15px 10px 0;
+	animation: isLoaded;
+	min-width: 550px; // for smaller screens
+	animation-duration: 250ms;
+
+	&.components-card {
+		box-shadow: none;
+		border: 1px solid $table-border;
+		border-radius: 8px;
+	}
+
+	.woocommerce-order-attribution-install-analytics-overview-banner__body {
+		display: flex;
+		align-items: center;
+	}
+
+	.woocommerce-order-attribution-install-analytics-overview-banner__image_container {
+		display: flex;
+		padding-left: 15px;
+		min-width: 200px; // for smaller screens
+	}
+
+	.woocommerce-order-attribution-install-analytics-overview-banner__text_container {
+		margin-inline: 24px;
+
+		> * {
+			margin-block: 1rem;
+		}
+
+		p {
+			margin-block: 0.1rem;
+		}
+	}
+
+	.woocommerce-order-attribution-install-analytics-overview-banner__text-chip {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		background: $studio-gray-0;
+		border-radius: 2px;
+		width: 42px;
+		height: 24px;
```

Comment:
WDYT about relacing 

```
display: inline-flex
justify-content: center;
background: $studio-gray-0;
border-radius: 2px;
padding: 2px 5px; 
```

So the "chip" container can adapt to the text.

Reply:
Also, I wonder if "chip" is the best name for this. I believe a wider used name may be "badge" or "label"?

Reply:
Thanks for the tip about styling. The "chip" is from the figma but I also like "badge" more.
Updated in https://github.com/woocommerce/woocommerce/pull/51672/commits/749fb4dd2a5a4e34933b14b888380db0bc8cf3b3#diff-71e16bc648210308ae9a1f2e30c8e8e311fe24425077449417a2f94ca8b2aa9bL37-R44